### PR TITLE
Fix issue with finalize when no percySnapshot has been called

### PR DESCRIPTION
**Pull request by mfazekas**
Fixes: #15 

Instead of `cannot read 'then' on undefined` we do  log error message with: `No percySnapshot calls were issued`

### CODE REVIEW

File: test/specs/e2e_spec.js [hunk 1]
```diff
@@ -227,6 +227,21 @@ describe('WDIO with percy', () => {
   });
 });
 
+describe('corner case: no perctSnapshot invocations', () => {
```
Comment (timhaines) [inline test/specs/e2e_spec.js]: Typo here, perctSnapshot => percySnapshot.

**Comment by timhaines** [comment]
@mfazekas Thanks for this!  If there are no snapshots and finalize is called, I think it would be great to log a warning, but not treat it as an error?  (so resolve rather than reject?)

This would be consistent with how Percy's other clients behave.
